orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/topology.xml.j2

```diff
@@ -201,7 +171,7 @@
     <service>
         <role>ZEPPELINWS</role>
         {% for hostloc in salt['pillar.get']('gateway:location')['ZEPPELIN_SERVER'] -%}
-        <url>ws://{{ hostloc }}:9995/ws</url>
+        <url>ws://{{ hostloc }}:{{ ports['ZEPPELIN'] }}</url>
```

Comment:
Need to keep the `/ws` on the end of this. Sorry didn't get to review this quicker.

Reply:
It should be `<url>ws://{{ hostloc }}:{{ ports['ZEPPELIN'] }}/ws</url>`

Reply:
Hm, I've tested this and worked without /ws

Reply:
I'll double check.

Reply:
@keyki and I checked this and looks like I was wrong and don't need the `/ws`.